import "response.proto";
import "serverinfo_user.proto";

message Response_Login {
    extend Response {
        optional Response_Login ext = 1005;
    }
    optional ServerInfo_User user_info = 1;
    repeated ServerInfo_User buddy_list = 2;
    repeated ServerInfo_User ignore_list = 3;
    optional string denied_reason_str = 4;
    optional uint64 denied_end_time = 5;
    repeated string missing_features = 6;
}
